src/pages/Explorer/Category/ExplorerCategory.svelte

```diff
@@ -79,45 +79,49 @@
     items = items
   })
 
-  function fetch(bypassLoading = false) {
+  async function setInsightItems() {
+    if (activeMenu !== MenuItem.TRENDING) return
+    const insightItems = await queryExplorerItems({
+      types: [EntityKeys.INSIGHT],
+      page: insightsPage,
+    })
+    insightsPages = insightItems.pages
+    insights = insightsPage === 1 ? insightItems.items : insights.concat(insightItems.items)
+  }
+
+  async function setDisplayingItems() {
+    const displayingItems = await queryExplorerItems({
```

Comment:
Usually, fetched data object is named `data`. Better to keep naming convention consistent